lib/git_reflow/git_helpers.rb

```diff
@@ -27,26 +27,41 @@ def get_first_commit_message
       run('git log --pretty=format:"%s" --no-merges -n 1', loud: false).strip
     end
 
-    def push_current_branch
-      run_command_with_label "git push origin #{current_branch}"
+    def push_current_branch(remote = 'origin')
```

Comment:
Use `options[:remote]` instead for the argument.  As a rule of thumb, if you can't tell what the argument should be without looking at the method definition, then it is better served in options.  As an example `fetch_destination` is pretty clear that the argument should be the destination branch.